bitrise.yml

```diff
@@ -1431,6 +1431,27 @@ workflows:
       bitrise.io:
         stack: osx-xcode-14.1.x-ventura
         machine_type_id: g2.8core
+  Bitrise-device-testing:
+    steps:
+    - cache-pull@2.1:
+        is_always_run: true
+    - xcode-build-for-test@2:
+        inputs:
+        - test_plan: Fennec_Enterprise_XCUITests
```

Comment:
I would use test_plan: Smoketest3 (or any number) so that we start by testing this with only a few tests and not the full test suite